rest/session_test.go

```diff
@@ -217,7 +217,7 @@ func TestLogin(t *testing.T) {
 	rt := NewRestTester(t, nil)
 	defer rt.Close()
 
-	a := auth.NewAuthenticator(rt.Bucket(), nil, auth.DefaultAuthenticatorOptions())
+	a := auth.NewAuthenticator(rt.Bucket().DefaultDataStore(), nil, auth.DefaultAuthenticatorOptions())
```

Comment:
This should be the metadata store associated with the 'db' being called below.  Should we add a MetadataStore() function to RestTester that returns dbCtx.MetadataStore from the default db?